src/geometry/manifold/mfderiv.lean

```diff
@@ -1232,6 +1232,70 @@ lemma mfderiv_within_const (hxs : unique_mdiff_within_at I s x) :
 
 end const
 
+section arithmetic
+/-! #### Arithmetic -/
+
+variables {S: topological_space.opens 𝕜}
+
+lemma mdifferentiable_add {f g : S → 𝕜} (hf : mdifferentiable 𝓘(𝕜) 𝓘(𝕜) f)
+  (hg : mdifferentiable 𝓘(𝕜) 𝓘(𝕜) g) : mdifferentiable 𝓘(𝕜) 𝓘(𝕜) (f + g) :=
```

Comment:
This should be called `mdifferentiable.add`, to take advantage of dot notation.  Can you please phrase it for functions into a `𝕜`-normed space, not just into `𝕜`?

Reply:
Also please add the `has_mfderiv_at` versions and deduce the `mdifferentiable` and `mfderiv` versions as a consequence.